src/utils/viewer/viewer.cpp

```diff
@@ -510,14 +544,23 @@ void Viewer::keyPressEvent(KeyEvent& event) {
                                          drawObjectBBs);
       }
     } break;
+    case KeyEvent::Key::One: {
+      displayPerformanceInfo = !displayPerformanceInfo;
```

Comment:
Bit trick, if you want: 

```suggestion
      displayPerformanceInfo ^= true;
```

Reply:
Hahaha. :)